fdbclient/DatabaseBackupAgent.actor.cpp

```diff
@@ -539,16 +541,25 @@ namespace dbBackup {
 		Future<Void> execute(Database cx, Reference<TaskBucket> tb, Reference<FutureBucket> fb, Reference<Task> task) { return _execute(cx, tb, fb, task); };
 		Future<Void> finish(Reference<ReadYourWritesTransaction> tr, Reference<TaskBucket> tb, Reference<FutureBucket> fb, Reference<Task> task) { return _finish(tr, tb, fb, task); };
 
-		ACTOR static Future<Void> dumpData(Database cx, Reference<Task> task, PromiseStream<RCGroup> results, FlowLock* lock, Reference<TaskBucket> tb) {
+		// store mutation data from results until the end of stream or the timeout. If breaks on timeout returns the first uncopied version
+		ACTOR static Future<Optional<Version>> dumpData(
+		  Database cx, Reference<Task> task, 
```

Comment:
I think our style is to use 4 spaces for indentation. See https://github.com/apple/foundationdb/blob/master/.clang-format

Reply:
Done